src/components/FlawFormOwner.vue

```diff
@@ -1,20 +1,28 @@
 <script setup lang="ts">
 import EditableText from '@/components/widgets/EditableText.vue';
 import { useUserStore } from '@/stores/UserStore';
-import { nextTick, computed } from 'vue';
+import { nextTick, computed, ref } from 'vue';
+
 const owner = defineModel<string | null>({ default: null });
 const userStore = useUserStore();
 
-function selfAssign() {
-  owner.value = userStore.userName;
+async function selfAssign() {
+  isLoading.value = true;
+  await userStore.cacheJiraUsername();
+  owner.value = userStore.jiraUsername;
```

Comment:
if `userStore.cacheJiraUsername()` fails, because it has a `try/catch` it will not throw and the `owner.value` will be set to `''`, removing the previous owner if there was any